projects/ngx-prx-styleguide/src/lib/charts/models/chart-type.type.ts

```diff
@@ -1 +1 @@
-export type ChartType = 'line' | 'pie' | 'area' | 'bar';
+export type ChartType = 'line' | 'spline' | 'pie' | 'area' | 'bar';
```

Comment:
Maybe add `spline` to the storybook types as well?

Reply:
Fixed and in before the sprint ended!